tests/test-cases/Group1-Docker-Commands/1-38-Docker-Exec.robot

```diff
@@ -123,3 +122,24 @@ Exec NonExisting
     #\   ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} exec ${id} /NonExisting
     #\   Should Be Equal As Integers  ${rc}  0
     #\   Should Contain  ${output}  no such file or directory
+
+Exec During PowerOff
+     ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} pull ${busybox}
+     Should Be Equal As Integers  ${rc}  0
+     Should Not Contain  ${output}  Error
+     ${rc}  ${id}=  Run And Return Rc And Output  docker %{VCH-PARAMS} run -d ${busybox} /bin/top -d 600
```

Comment:
don't think you need -d 600 here.. no reason to create a race here, just let it run